app/views/root/index.html.erb

```diff
@@ -173,10 +169,10 @@
           <div class="departments-promo-sections">
             <div class="promo-image">
               <div class="promo-content">
-                <a href="/government/topical-events/eu-referendum"><%= image_tag 'homepage/eu-referendum.jpg', alt: 'EU referendum' %></a>
-                <h3>EU referendum</h3>
+                <a href="/jobsearch"><%= image_tag 'homepage/find-a-job.png', alt: 'Find a job' %></a>
+                <h3>Find a job</h3>
                 <p>
-                  <a href="/government/topical-events/eu-referendum">Vote on Thursday 23 June</a> and help to decide if the UK remains part of the European Union.
+                  Use <a href="/jobsearch">Universal Jobmatch</a> to find full or part-time work in Great Britain and abroad.
```

Comment:
In terms of a11y do we want to duplicate links?

Think it'd be better to reduce the amount of duplicate links, could consider putting `aria-hidden`?

cc @alphagov/accessibility